src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -1203,59 +1191,6 @@ def _to_snowpark_dataframe_from_snowpark_pandas_dataframe(
             col_mapper=rename_mapper
         )
 
-    def to_csv_with_snowflake(self, **kwargs: Any) -> None:
```

Comment:
seems like these query compiler deletions are a merge artifact, can you double check @sfc-gh-mvashishtha?

Reply:
Yep, removed